frontend/public/locales/en/common.json

```diff
@@ -129,5 +131,51 @@
     "languages": "Languages",
     "languages-summary": "Choose the language you want to see the website in.",
     "languages-description": "Select a language to switch to it:",
-    "switch-language": "Switch Language"
+    "switch-language": "Switch Language",
+    "actions": "Actions",
+    "card-expiry": "Expiry: {{month}} / {{year}}",
+    "checkout": "Checkout",
+    "confirm-selection": "Confirm Selection",
+    "created": "Created",
+    "id": "ID",
+    "loading-saved-payment-methods": "Loading saved payment methods…",
+    "payment": "Payment",
+    "make-donation": "Make Donation",
+    "no-saved-payment-methods": "No saved payment methods to show.",
+    "no-transaction-history": "No transaction history to show.",
+    "payment-summary": "Payment Summary",
+    "remove-saved-card": "Remove Saved Card",
+    "retry-checkout": "Retry Checkout",
+    "save-card-for-reuse": "Save card for reuse",
+    "saved-cards": "Saved Cards",
+    "select-donation-amount": "Select an amount to donate",
+    "status": "Status",
+    "stripe-receipt": "Stripe receipt",
+    "submit-payment": "Submit payment",
+    "transaction-breakdown": "Transaction Breakdown",
+    "transaction-cancel": "Cancel Transaction",
+    "transaction-cancelled": "Transaction cancelled",
+    "transaction-history": "Transaction History",
+    "transaction-summary": "Transaction Summary",
+    "transaction-summary-created": "Date created: {{date}}",
+    "transaction-summary-updated": "Last updated: {{date}}",
+    "transaction-summary-id": "ID: {{id}}",
```

Comment:
Maybe we should not abbreviate this? So just say `Identifier: {{id}}`

Reply:
Good shout, landed on ID out of habit, but unabbreviated is also entirely unambiguous.